packages/e2e-tests/tests/object_preview-01.test.ts

```diff
@@ -3,137 +3,80 @@ import {
+  await waitForConsoleMessage(screen, "ƒbar()");
+
+  await waitForConsoleMessage(screen, "Proxy{}");
   await waitForConsoleMessage(screen, "Symbol()");
   await waitForConsoleMessage(screen, "Symbol(symbol)");
-  await waitForConsoleMessage(screen, `Object { "Symbol()": 42, "Symbol(symbol)": "Symbol()" }`);
-
-  msg = await waitForConsoleMessage(screen, "Object { _foo: C }");
+  await waitForConsoleMessage(screen, `{Symbol(): 42, Symbol(symbol): Symbol()}`);
 
-  // TODO: add support for expanding objects in the console
-  await checkMessageObjectContents(
-    screen,
-    msg,
-    ['foo: C { baz: "baz" }', 'bar: "bar"', 'baz: "baz"'],
-    ["foo", "bar"]
-  );
+  await waitForConsoleMessage(screen, "{_foo: C{…}, foo: ƒ()}");
 
   await warpToMessage(screen, "Done");
 
   await executeInConsole(screen, "Error('helo')");
-  await waitForConsoleMessage(screen, 'Error: "helo"');
+  await waitForConsoleMessage(screen, "Error: helo");
 
-  await executeInConsole(
-    screen,
-    `
-        function f() {
-          throw Error("there");
-        }
-        f();
-      `
-  );
+  await executeInConsole(screen, 'function f() { throw Error("there"); }()');
   // FIXME the first line in this stack isn't right.
-  await waitForConsoleMessage(screen, 'Error: "there"');
+  await waitForConsoleMessage(screen, "Error: there");
 
-  executeInConsole(screen, "Array(1, 2, 3)");
-  msg = await waitForConsoleMessage(screen, "Array(3) [ 1, 2, 3 ]");
-  await checkMessageObjectContents(screen, msg, ["0: 1", "1: 2", "2: 3", "length: 3"]);
+  await executeInConsole(screen, "Array(1, 2, 3)");
+  await waitForConsoleMessage(screen, "(3) [1, 2, 3]");
 
   await executeInConsole(screen, "new Uint8Array([1, 2, 3, 4])");
-  msg = await waitForConsoleMessage(screen, "Uint8Array(4) [ 1, 2, 3, 4 ]");
-  await checkMessageObjectContents(screen, msg, ["0: 1", "1: 2", "2: 3", "3: 4", "length: 4"]);
+  await waitForConsoleMessage(screen, "Uint8Array(4) [1, 2, 3, 4]");
 
   await executeInConsole(screen, `RegExp("abd", "g")`);
-  msg = await waitForConsoleMessage(screen, "/abd/g");
-
-  // RegExp object properties are not currently available in chromium.
-
-  await checkMessageObjectContents(screen, msg, ["global: true", `source: "abd"`]);
+  await waitForConsoleMessage(screen, "/abd/g");
 
   await executeInConsole(screen, "new Set([1, 2, 3])");
-  msg = await waitForConsoleMessage(screen, "Set(3) [ 1, 2, 3 ]");
-  await checkMessageObjectContents(screen, msg, ["0: 1", "1: 2", "2: 3", "size: 3"], ["<entries>"]);
+  await waitForConsoleMessage(screen, "Set(3) [1, 2, 3]");
 
   await executeInConsole(screen, "new Map([[1, {a:1}], [2, {b:2}]])");
-  msg = await waitForConsoleMessage(screen, "Map { 1 → {…}, 2 → {…} }");
-  await checkMessageObjectContents(
-    screen,
-    msg,
-    ["0: 1 → Object { a: 1 }", "1: 2 → Object { b: 2 }", "size: 2"],
-    ["<entries>"]
-  );
+  await waitForConsoleMessage(screen, "Map(2) {1 → {…}, 2 → {…}}");
```

Comment:
We could probably just do e.g. "Map(2)" for this type of object, but no big deal 😁